apps/examples/control_log/ctrlog_usage.h

```diff
@@ -0,0 +1,44 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
```

Comment:
Please change to 2022.

Reply:
Thannks I'll modify it. Actually it's written end of 2021 :)